### PR TITLE
[Security] Add changelog note about calling the constructor of `DefaultLoginRateLimiter` with an empty secret

### PR DESCRIPTION
**Pull request by alexandre-daubois**
| Q             | A
| ------------- | ---
| Branch?       | 6.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | yes
| Tickets       | -
| License       | MIT
| Doc PR        | -

Related to https://github.com/symfony/symfony/pull/51550

### CODE REVIEW

File: src/Symfony/Component/Security/Http/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ---
 
  * `UserValueResolver` no longer implements `ArgumentValueResolverInterface`
+ * Calling the constructor of `DefaultLoginRateLimiter` with an empty secret is deprecated
```
Comment (derrabus) [inline src/Symfony/Component/Security/Http/CHANGELOG.md]:
```suggestion
 * Deprecate calling the constructor of `DefaultLoginRateLimiter` with an empty secret
```

**Review by derrabus — CHANGES_REQUESTED** [review]
Please also update UPGRADE-6.4.md accordingly.

File: src/Symfony/Component/Security/Http/CHANGELOG.md (hunk 1 above)
Comment (alexandre-daubois) [reply src/Symfony/Component/Security/Http/CHANGELOG.md]: Updated :+1: I'm not exactly sure how to word this [in 7.0](https://github.com/symfony/symfony/pull/51550/files#diff-6d49df895edd288cee25c95df434ae329795980f8024cdeb33ccd66ab65fb52cR306) :thinking:

**Comment by alexandre-daubois** [comment]
Status: Needs review

**Review by javiereguiluz — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @alexandre-daubois.

**Pull request merged**